src/index.ts

```diff
@@ -1,5 +1,16 @@
-import { NameGenerator } from './name-generator'
+import { ProxyServer } from './interfaces/proxy-server'
+import { FastifyServer } from './fastify-server'
+import { ConsoleLogger } from './console-logger'
+import { Logger } from './interfaces/logger'
 
-const nameGenerator: NameGenerator = new NameGenerator()
+const SOFIE_REST_URL: string = 'http://localhost:3005'
```

Comment:
*Cough cough*, might we want different names for the variables? Or would we rather come back to that?

Reply:
I would rather that we do it when the repo and package is renamed to avoid an in-between state.